test/player_external.js

```diff
@@ -70,10 +104,14 @@ describe('Player', () => {
 
     await player.destroy();
     player.releaseAllMutexes();
+    shaka.util.Dom.removeAllChildren(adContainer);
   });
 
   afterAll(() => {
+    document.head.removeChild(imaScript);
+    document.head.removeChild(daiScript);
```

Comment:
I did a quick test, and removing a script element does not remove anything that script did, including exporting functions and classes to `window`.  So if IMA is loaded here, then removed, then loaded again, it's no different from IMA being loaded twice without the removal.  If loading the IMA script twice causes a failure in IMA, we can't fix it by removal.

I think we need to take scripts like this and consolidate them to test boot time.

There are also several places probeSupport() is used, which is slow, so maybe probeSupport() should also be called only once at boot and the results stored globally for use in all tests.

I'd like to see those changes first, in a separate PR, then fix the external tests using those changes.

Reply:
IMA --> https://github.com/shaka-project/shaka-player/pull/6666

Reply:
probeSupport --> https://github.com/shaka-project/shaka-player/pull/6667

Reply:
Done!